client/packages/inventory/src/Stocktake/DetailView/ReduceLinesToZeroModal.tsx

```diff
@@ -68,7 +68,7 @@ export const ReduceLinesToZeroConfirmationModal = ({
               value={reason}
               onChange={reason => setReason(reason)}
               reasonOptions={reasonOptions?.nodes ?? []}
-              isLoading={isLoading}
+              loading={isLoading}
```

Comment:
Can't comment up above since I haven't made a change there... but ReasonSearchInput can be refactored further, so that we don't need to pass in reasonOptions and loading... We made a decision to show this everywhere and just have it disabled if it isn't required, so was there a reason why it was hidden here?